ssl/ssl_lib.cc

```diff
@@ -3311,33 +3310,16 @@ int SSL_set1_curves_list(SSL *ssl, const char *curves) {
 }
 
 size_t SSL_client_hello_get0_ciphers(SSL *ssl, const unsigned char **out) {
-  STACK_OF(SSL_CIPHER) *client_cipher_suites = SSL_get_client_ciphers(ssl);
-  if (client_cipher_suites == nullptr) {
+  if (SSL_get_client_ciphers(ssl) == nullptr) {
       return 0;
   }
+  const char * ciphers = ssl->all_client_cipher_suites.get();
+  if (ciphers == nullptr) {
```

Comment:
It shouldn't possible for this to be null if the logic is correct, right?  Could we put an assert here to make such a failure more obvious during testing:
```c++
assert!(ciphers != nullptr);
```

Reply:
Yep, made the change!